DarynaGrytsaieva/src/main/java/Test.java

```diff
@@ -1,5 +1,5 @@
 public class Test {
     public static void main(String[] args) {
-        System.out.println("Hello!");
+        System.out.println("Hello world!");
```

Comment:
Why is the "world" word without capital letter?

Reply:
Because I want